index.html

```diff
@@ -5,6 +5,10 @@
     <link rel="stylesheet" href="css/application.css">
   </head>
   <body>
+    <nav class="navbar">
+      <img class="wolox-logo" src="assets/wolox_logo.svg">
+      <h1>Developer Tools</h1>
```

Comment:
Add class to the title.
All html tags, if they have styles, should have a class.